vizro-core/changelog.d/20240515_133813_petar_pejovic_fixing_dash_table_dropdown_menu.md

```diff
@@ -0,0 +1,49 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Added
+
+- A bullet item for the Added category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Changed
+
+- Fixing `dash_data_table` dropdown menu to open on click. ([#481](https://github.com/mckinsey/vizro/pull/481))
```

Comment:
```suggestion
- Fix `dash_data_table` dropdown menu to open on click. ([#481](https://github.com/mckinsey/vizro/pull/481))
```
Remember the tense is given in the examples in this file: "Enable feature XXX". It should ideally be imperative, like the common convention with commit messages (that we don't really care about).